ci/travis_before_script_cpp.sh

```diff
@@ -36,7 +36,7 @@ while true; do
     esac
 done
 
-if [ "$only_library_mode" == "no" ]; then
+if [ $only_library_mode == "no" ]; then
```

Comment:
Shouldn't the quotes stay here as-is?

Reply:
It's my mistake.  I reverted this change.  Thanks for catching this.